imap_processing/codice/constants.py

```diff
@@ -78,112 +78,271 @@
 # TODO: Hi products shape should be energy x ssd index x spin sector (8*12*12)
 DATA_PRODUCT_CONFIGURATIONS = {
     CODICEAPID.COD_HI_INST_COUNTS_AGGREGATED: {
+        "coords": [
+            "epoch",
+            "inst_az",
+            "spin_sector",
+            "esa_step",
+            "energy_label",
+        ],  # TODO: These will likely change
+        "dataset_name": "imap_codice_l1a_hi_counters_aggregated",
+        "dims": [
+            "epoch",
+            "esa_step",
+            "inst_az",
+            "spin_sector",
+        ],  # TODO: These will likely change
+        "instrument": "hi",
         "num_counters": 1,
         "num_energy_steps": 1,  # TODO: Double check with Joey
         "num_positions": 6,  # TODO: Double check with Joey
         "num_spin_sectors": 1,
+        "support_variables": [],  # TODO: Double check with Joey
         "variable_names": HI_INST_COUNTS_AGGREGATED_NAMES,
-        "dataset_name": "imap_codice_l1a_hi_counters_aggregated",
-        "instrument": "hi",
     },
     CODICEAPID.COD_HI_INST_COUNTS_SINGLES: {
+        "coords": [
+            "epoch",
+            "inst_az",
+            "spin_sector",
+            "esa_step",
+            "energy_label",
+        ],  # TODO: These will likely change
+        "dataset_name": "imap_codice_l1a_hi_counters_singles",
+        "dims": [
+            "epoch",
+            "esa_step",
+            "inst_az",
+            "spin_sector",
+        ],  # TODO: These will likely change
+        "instrument": "hi",
         "num_counters": 3,
         "num_energy_steps": 1,  # TODO: Double check with Joey
         "num_positions": 16,  # TODO: Double check with Joey
         "num_spin_sectors": 1,
+        "support_variables": [],  # No support variables for this one
         "variable_names": HI_INST_COUNTS_SINGLES_NAMES,
-        "dataset_name": "imap_codice_l1a_hi_counters_singles",
-        "instrument": "hi",
     },
     CODICEAPID.COD_HI_OMNI_SPECIES_COUNTS: {
+        "coords": [
+            "epoch",
+            "inst_az",
+            "spin_sector",
+            "esa_step",
+            "energy_label",
+        ],  # TODO: These will likely change
+        "dataset_name": "imap_codice_l1a_hi_omni",
+        "dims": [
+            "epoch",
+            "esa_step",
+            "inst_az",
+            "spin_sector",
+        ],  # TODO: These will likely change
+        "instrument": "hi",
         "num_counters": 8,
         "num_energy_steps": 15,  # TODO: Double check with Joey
         "num_positions": 4,  # TODO: Double check with Joey
         "num_spin_sectors": 1,
+        "support_variables": ["data_quality", "spin_period"],
```

Comment:
Does this mean that there are these two extra data variable stored in CDF file and others only has science data stored?